docs/contributing/committing.md

```diff
@@ -15,84 +14,81 @@ Most code contributions will be in the form of pull requests from Github.
 which can be merged with the 'Squash and merge' option. Instead, the code should
 be checked out by a committer locally, the changes examined and rebased,
-the ``CHANGELOG.txt`` and release notes updated,
-and finally the code should be pushed to the ``main`` branch.
+the `CHANGELOG.txt` and release notes updated,
+and finally the code should be pushed to the `main` branch.
 This process is covered in more detail below.
 
-Check out the code locally
-==========================
+## Check out the code locally
 
 If the code has been submitted as a pull request,
 you should fetch the changes and check them out in your Wagtail repository.
-A simple way to do this is by adding the following ``git`` alias to your ``~/.gitconfig`` (assuming ``upstream`` is ``wagtail/wagtail``):
-
-.. code-block:: text
+A simple way to do this is by adding the following `git` alias to your `~/.gitconfig` (assuming `upstream` is `wagtail/wagtail`):
 
-    [alias]
-        pr = !sh -c \"git fetch upstream pull/${1}/head:pr/${1} && git checkout pr/${1}\"
+```text
+[alias]
+    pr = !sh -c \"git fetch upstream pull/${1}/head:pr/${1} && git checkout pr/${1}\"
+```
 
-Now you can check out pull request number ``xxxx`` by running ``git pr xxxx``.
+Now you can check out pull request number `xxxx` by running `git pr xxxx`.
 
-Rebase on to ``main``
-=====================
+## Rebase on to `main`
 
-Now that you have the code, you should rebase the commits on to the ``main`` branch.
+Now that you have the code, you should rebase the commits on to the `main` branch.
 Rebasing is preferred over merging,
 as merge commits make the commit history harder to read for small changes.
 
 You can fix up any small mistakes in the commits,
 such as typos and formatting, as part of the rebase.
-``git rebase --interactive`` is an excellent tool for this job.
+`git rebase --interactive` is an excellent tool for this job.
 
 Ideally, use this as an opportunity to squash the changes to a few commits, so
 each commit is making a single meaningful change (and not breaking anything).
 If this is not possible because of the nature of the changes, it's acceptable
 to either squash into a commit or leave all commits unsquashed,
 depending on which will be more readable in the commit history.
 
-.. code-block:: console
-
-    $ # Get the latest commits from Wagtail
-    $ git fetch upstream
-    $ git checkout main
-    $ git merge --ff-only upstream/main
-    $ # Rebase this pull request on to main
-    $ git checkout pr/xxxx
-    $ git rebase main
-    $ # Update main to this commit
-    $ git checkout main
-    $ git merge --ff-only pr/xxxx
-
-Update ``CHANGELOG.txt`` and release notes
-==========================================
-
-.. note::
-
-    This should only be done by core committers, once the changes have been reviewed and accepted.
-
-Every significant change to Wagtail should get an entry in the ``CHANGELOG.txt``,
+```console
+$ # Get the latest commits from Wagtail
```

Comment:
Can we please get rid of the leading `$` on the console code blocks. 

We are moving away from that convention as it makes it harder to copy/ paste the commands.